fedora/fas.py

```diff
@@ -178,11 +178,11 @@ async def user_hello(self, evt: MessageEvent, username: str | None) -> None:
         """
         await self._user_hello(evt, username)
 
-    @user.subcommand(name="info", help="Return brief information about a Fedora user.")
+    @user.subcommand(name="info", help="Return detailed information about a Fedora user, including username, human name, pronouns, creation date, timezone, locale, and GPG key IDs.")
```

Comment:
Unless pull requests depend on other pull requests, it's typical to not include changes already in a pull request. I'm guessing this was not intentional since this is very easy to do with git, unfortunately.

It should resolve itself as long as #74 gets merged first, but for future reference or if you want to fix it yourself, one way to do it (there are lots of ways, but this is my preference) is use `git-rebase`: `git checkout Issue-71-Fix && git rebase -i main`. This will put you in an editor where every line is a commit being rebased. Deleting lines will remove those commits from your branch and you can also re-order commits this way. Removing the line with commit 24b610e3ae696a371f6db30cc857502bf358e343 should fix it.

This will mean that to push the branch you will need to use the `-f` flag, which is something to be careful with since it lets you overwrite changes, which is what we want in this case. It is also easy to lose work this way.